src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ERR.cs

```diff
@@ -2,25 +2,26 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Diagnostics;
 using System.Runtime.InteropServices;
 using System.Security.Cryptography;
 using System.Text;
 
 internal static partial class Interop
 {
-    internal static partial class libcrypto
+    internal static partial class Crypto
     {
         [DllImport(Libraries.LibCrypto)]
-        private static extern uint ERR_get_error();
+        private static extern ulong ErrGetError();
 
         [DllImport(Libraries.LibCrypto, CharSet = CharSet.Ansi)]
-        private static extern void ERR_error_string_n(uint e, StringBuilder buf, int len);
+        private static extern void ErrErrorStringN(ulong e, StringBuilder buf, int len);
```

Comment:
(That said, if we care about performance here, I think we'd change this to just stackalloc a byte*, change the signature to take a byte* instead of a StringBuilder, and then use Marshal.PtrToStringAnsi to extract the string.)